NVSHAS-9890 [Documentation] Need to add note for Zero-drift Process Protection (#164)
neuvector/docs: docs/05.policy/06.processrules/06.processrules.md

This is the default mode for process and file protections. Zero-drift automatically allows only processes which originate from the parent process that is in the original container image, and does not allow file updates or new files to be installed. When in Discover or Monitor mode, zero-drift will alert on any suspicious process or file activity. In Protect mode, it will block such activity. Zero-drift does not require processes to be learned or added to an allow-list. Disabling zero-drift for a group will cause the process and file rules listed for the group to take effect instead.

:::note
(1) The process/file rules listed for each group are always applied, even when zero-drift is enabled. This offers a way to add allow/deny exceptions to the base zero-drift protections. Keep in mind that if a group starts in Discover mode, process/file rules can be automatically added to the list, and should be reviewed and edited before moving to Monitor/Protect modes.

> (2) Its recommnded to set system-trusted (de-facto) namespaces (system and network-plugin) as BASIC protection. For example, "kube-system", "openshift-ovn-kubernetes", "ovn-*" etc. and set other application pods as Zero Drift.

Reviewer: ```suggestion
(2) It's recommended to set system-trusted (de-facto) namespaces (system and network-plugin) as BASIC protection. Example namespaces to set are "kube-system", "openshift-ovn-kubernetes", "ovn-*" etc. Set other application pods as Zero Drift.
```
